lib/application/crowdaction/subscription/status/subscription_status_bloc.dart

```diff
@@ -0,0 +1,46 @@
+import 'dart:async';
```

Comment:
Should we move some files around a bit?
`application/crowdaction/subscription_status/`
`application/crowdaction/subscription/`
Just a suggestion!

Reply:
Like keep the blocs in the same folder ??

Reply:
No, but right now you have it like this:

lib/application/crowdaction/subscription/
  - bloc_files
  - /status/
    - bloc_files

And it should be:
lib/application/crowdaction/subscription/
  - bloc_files
lib/application/crowdaction/subscription_status/
  - bloc_files

Reply:
You have a folder with a bloc and inside that bloc's folder you have a folder to another bloc, and they aren't related to each other logic wise, there's no dependency.